docs/standard/serialization/system-text-json/overview.md

```diff
@@ -44,22 +44,10 @@ For information about security threats that were considered when designing <xref
 
 The `System.Text.Json` types are thread-safe, including:
```

Comment:
This is a strong a statement and a false one at that. We might want to replace the entire section with something saying that the serializer of `System.Text.Json` has been designed with thread safety in mind. Practically this means that `JsonSerializerOptions` instances once locked can be safely shared across multiple threads.